pkg/update_pod_status_util.go

```diff
@@ -597,6 +598,32 @@ func (initp *InitParams) manageVolumesAndUpdateStatus(ctx context.Context, resta
 	metadata.InitializedVolumes = initializedVolumes
 	metadata.DirtyVolumes = dirtyVolumes
 
+	data, err := os.ReadFile(aerospikeConf)
+	if err != nil {
+		return err
+	}
+
+	if pod.Annotations == nil {
+		pod.Annotations = make(map[string]string)
+	}
+
+	pod.Annotations["aerospikeConf"] = string(data)
+
+	if err := retry.OnError(retry.DefaultBackoff, func(err error) bool {
+		// Customize the error check for retrying, return true to retry, false to stop retrying
+		return true
+	}, func() error {
+		// Patch the resource
+		if err := initp.k8sClient.Update(ctx, pod); err != nil {
```

Comment:
I think we should get the latest object first here to fix object already modified errors also

Reply:
Above annotations code will also go inside retry func